lib/classes/Swift/Mime/Headers/AbstractHeader.php

```diff
@@ -234,15 +234,12 @@ protected function setFieldName($name)
     /**
      * Produces a compliant, formatted RFC 2822 'phrase' based on the string given.
      *
-     * @param Swift_Mime_Header        $header
-     * @param string                   $string  as displayed
-     * @param string                   $charset of the text
-     * @param Swift_Mime_HeaderEncoder $encoder
-     * @param bool                     $shorten the first line to make remove for header name
+     * @param string $string  as displayed
+     * @param bool   $shorten the first line to make remove for header name
      *
      * @return string
      */
-    protected function createPhrase(Swift_Mime_Header $header, $string, $charset, Swift_Mime_HeaderEncoder $encoder = null, $shorten = false)
+    protected function createPhrase($string, $shorten = false)
```

Comment:
You need to keep the signature as changing it would be a BC break. We can then clean that for 6.0.